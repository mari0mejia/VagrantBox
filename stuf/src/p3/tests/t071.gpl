int a = !7;
int b = !12.5;
double d = !7;
double e = !12.5;
string g = !7;
string h = !12.5;
