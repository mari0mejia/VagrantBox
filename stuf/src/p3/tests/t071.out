gpl.cpp::main()
  input file(tests/t071.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
int a = 0
int b = 0
double d = 0
double e = 0
string g = "0"
string h = "0"
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
